include/lsst/afw/image/ImageExceptions.h.m4

```diff
@@ -9,13 +9,11 @@ dnl comment to go into the output file
 // This file is machine generated from Runtime.h.m4. Please do not edit directly
 //
 dnl
-/** @file
-  * @brief LSST exception handler classes for Image exceptions
-  *
-  * There are currently no afw::image exceptions.  If there ever are, we
-  * need to refactor the lsst::lex::exceptions code (maybe the m4 file)
-  * to allow us to define new Exceptions without repeating that boilerplate
-  *
-  * @ingroup afw
-  */
+/*
+ * LSST exception handler classes for Image exceptions
+ *
+ * There are currently no afw::image exceptions.  If there ever are, we
+ * need to refactor the lsst::lex::exceptions code (maybe the m4 file)
+ * to allow us to define new Exceptions without repeating that boilerplate
+ */
```

Comment:
What about the `Runtime.h.m4` the comment mentions? Is that something we no longer have?